TaskListRails/app/controllers/tasks_controller.rb

```diff
@@ -7,4 +7,23 @@ def show
     id = params[:id]
     @task = Task.find(id)
   end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    Task.create(task_params[:task])
+    redirect_to "/"
+  end
+
+  def destroy
+    Task.destroy(params[:id])
+    redirect_to "/"
+  end
+  private
```

Comment:
Best to have one line of whitespace on either side (above & below)of this private keyword